app/connectors/NsiConnector.scala

```diff
@@ -94,7 +102,9 @@ class NsiConnector @Inject() (
     s"$domain$rootPath$resourcePath$pathParams"
   }
 
-  private val CORRELATION_ID = servicesConfig.getString(s"microservice.services.$serviceName.correlationIdHeader")
+  private val CORRELATION_ID   = servicesConfig.getString(s"microservice.services.$serviceName.correlationIdHeader")
+  private val AUTHORIZATION    = "Authorization"
```

Comment:
The reason `CORRELATION_ID` is a static constant is because it's a custom header. Following my previous recommendation, you can delete this line.